File: 0058-length-of-last-word/0058-length-of-last-word.cs
public class Solution {
    public int LengthOfLastWord(string s) {

         int pointer = 0, len = 0;

        for (int i = s.Length - 1; i >= 0; --i)
            if (s[i] != ' '){ 
                pointer = i;
                break;
             }

        for (int i = pointer; i >= 0; --i){
            if (s[i] != ' ') 
                len++;
            else
                break;
        }    
             
        return len;

        
        /*
        string[] split = s.Split(new char [] { ' ' }, StringSplitOptions.RemoveEmptyEntries);
        return split[split.Length - 1].Length;
        */
    }
}

File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        
        int sizeS = s.size() - 1;
        int end;
        for (int i = sizeS; i >= 0; --i){
            if (s[i] == ' ')
                continue;
            else{
                end = i;
                break;
            }
        }
        vector<char> c;
        for (int i = end; i >= 0; --i){
            if (s[i] != ' ')
                c.push_back(s[i]);
            else 
                break;
        }
        return c.size();
    }
};